libr/anal/p/anal_dalvik.c

```diff
@@ -1157,6 +1720,7 @@ RAnalPlugin r_anal_plugin_dalvik = {
 	.bits = 32,
 	.desc = "Dalvik (Android VM) bytecode analysis plugin",
 	.op = &dalvik_op,
+	.opasm = &dalvik_opasm,
```

Comment:
Might want to comment out the opasm for the time being since its basically unimplemented

Reply:
yeah its probably only useful for "nop" and maybe another 0 argument opcode